apidoc/validate.py

```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Do you happen to know off-hand what the additional checks are? Or do we need to compare the two file and try and figure it out from there?